Customizing/global/plugins/Services/Repository/RepositoryObject/TalentAssessment/sql/dbupdate.php

```diff
@@ -88,4 +88,23 @@
 		."SET title = 'pl_xtas_observator'\n"
 		."WHERE title = 'il_xtas_observator'";
 $ilDB->query($query);
+?>
+
+<#10>
+<?php
+global $ilUser,$ilDB;
+$b = new \CaT\Plugins\CareerGoal\Settings\ilDB($ilDB, $ilUser);
+$settings_db = new \CaT\Plugins\TalentAssessment\Settings\ilDB($ilDB, $ilUser, $b);
+$settings_db->install();
+?>
```

Comment:
Weil addColumns protected ist und in install nur sachen angelegt werden, die schon exestieren.